src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -95,6 +95,23 @@ def export_config_outputs(self) -> Dict[str, Any]:
         """
         return {}
 
+    def tool_config_prefix(self) -> str:
+        """
+        Returns the config string that contains all tool specific settings
+
+        :return: A string that is the prefix for all tool specific settings
+        """
+        return ""
```

Comment:
Maybe we can make this method abstract?

Reply:
Sure. I wasn't exactly sure what that annotation did.

Reply:
Sounds good. Also if we make it abstract we can just do `pass` instead of `return ""` in the body.